media-libs/opencolorio/opencolorio-1.1.0-r1.ebuild

```diff
@@ -45,8 +45,11 @@ PATCHES=(
 	"${FILESDIR}/${P}-use-GNUInstallDirs-and-fix-cmake-install-location.patch"
 	"${FILESDIR}/${P}-remove-building-of-bundled-programs.patch"
 	"${FILESDIR}/${P}-yaml-cpp-0.6.patch"
+	"${FILESDIR}/${P}-remove-Werror.patch"
 )
 
+S="${WORKDIR}/OpenColorIO-${PV}"
```

Comment:
why is this needed now?

Reply:
Hmm... looks like it's not. I was probably thinking this is needed because of the different cases in the work subfolder and the name of the build.
I remove it from both ebuilds. Thanks for the hint!

Reply:
Exactly right, -r1 didn't need it because of `vcs-snapshot`.